app/views/publishers/vacancies/build/about_the_role.html.slim

```diff
@@ -0,0 +1,63 @@
+- content_for :page_title_prefix, page_title_prefix(vacancy, form, t("publishers.vacancies.steps.about_the_role"))
+
+= vacancy_form_page_heading(vacancy, step_process)
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for form, url: wizard_path(current_step, back_to: params[:back_to]), method: :patch do |f|
+      = f.govuk_error_summary
+
+      - if vacancy.job_role == "teacher"
+        = f.govuk_collection_radio_buttons :ect_status,
+          Vacancy.ect_statuses.keys,
+          :to_s,
+          classes: ["ect-status-radios"],
+          legend: { size: "m" }
+
+      = f.govuk_text_area :skills_and_experience,
+                          label: -> { tag.label(t("helpers.label.publishers_job_listing_about_the_role_form.skills_and_experience"), class: ["govuk-label--m", "govuk-!-font-weight-bold"]) },
```

Comment:
seems odd these govuk classes have to be applied, i think there are arguments you could pass to acheive this (and possibly a few below in this same file. `label: { size: "m" }` for example